### ISSUE TITLE
Add ability to bind servlets when using embedded Jetty.

### ISSUE THREAD

**Issue by Crias** [perwendel/spark#540]
(_Unless I've missed it - I've scoured the documentation and haven't found anything yet._)

When integrating an application with other libraries, sometimes those libraries provide features via servlets. For example, [Togglz](http://www.togglz.org/) provides a management console via a servlet.

When running in an embedded mode, there is no `web.xml` to add the appropriate servlet binding to.

My ideal usage would be something like:

`servlet("/togglz/*", org.togglz.console.TogglzConsoleServlet.class)`

As an equivalent to:

```
<servlet>
  <servlet-name>TogglzConsoleServlet</servlet-name>
  <servlet-class>org.togglz.console.TogglzConsoleServlet</servlet-class>
</servlet>
<servlet-mapping>
  <servlet-name>TogglzConsoleServlet</servlet-name>
  <url-pattern>/togglz/*</url-pattern>
</servlet-mapping>
```

**Comment by tipsy** [comment]
@Crias would #813 fix this for you?

**Comment by Crias** [comment]
I'm no longer working on the project that required this, but at a glance
I'm fairly confident this would fix the issue I had. This can be closed off
- thanks for checking back.

**Comment by StefanHub** [comment]
I need help. How would that be accomplished with the fix #813?
